README.md

```diff
@@ -127,6 +127,8 @@ $ bazel test test:eventuals
 
 ## User Guide
```

Comment:
Let's actually move [Building/Testing](https://github.com/3rdparty/eventuals#buildingtesting) and [Visual Studio Code and Bazel Set Up](https://github.com/3rdparty/eventuals#visual-studio-code-and-bazel-set-up) down to the [Contributing](https://github.com/3rdparty/eventuals#contributing) section. That way `User Guide` will come first.